third-culture-software/bhima
# Move reference_number from inventory to lot

Moved the column 'reference_number' from the inventory table to the lot table (per discussion in the issue).

Closes https://github.com/IMA-WorldHealth/bhima/issues/6433

**TESTING**
- Try editing an inventory item and notice that 'reference_number' is no longer available.
- Go to Stock > Articles in stock and pick an inventory item that has lots
- Edit the inventory item and change it to an asset and set the manufacturer brand/model
- Find the lots for the inventory item and edit one of them.  Set the reference_number
- In the Stock > Lots registry change the column definitions so show the asset related fields (is_asset, manufacturer_brand, manufacturer_model, serial_number) and then SAVE the column definitions
- Notice the asset related fields are available
- Use the Search and enter the reference number just added and notice that only that lot is shown
- Clear the search filter to revert to showing all lots
- Do a similar search filter test for is_asset
